fastapi_utils/crud/route.py

```diff
@@ -0,0 +1,107 @@
+from typing import ClassVar, Dict, Generic, Tuple, TypeVar
+
+from fastapi import Depends, HTTPException
+from pydantic import BaseModel
+from sqlalchemy.orm import Session
+
+from fastapi_utils.crud import CRUDBase
+
+ResponseModelType = TypeVar("ResponseModelType", bound=BaseModel)
+ResponseModelManyType = TypeVar("ResponseModelManyType", bound=BaseModel)
+CreateSchemaType = TypeVar("CreateSchemaType", bound=BaseModel)
+UpdateSchemaType = TypeVar("UpdateSchemaType", bound=BaseModel)
+CRUDBaseType = TypeVar("CRUDBaseType", bound=CRUDBase)
```

Comment:
Do you know how to get mypy to detect the crud_base would have the get_many, get, delete methods?